lib/Format.php

```diff
@@ -267,7 +267,7 @@ public static function guessFormat($data, $filename = null)
         }
 
         // Then try and guess by the first 1024 bytes of content
-        $short = substr($data, 0, 1024);
+        $short = substr($data ?? '', 0, 1024);
```

Comment:
One of the main PHP 8.1 is that built-in function that use to accept NULLs in string arguments will now trigger a deprecation.